src/js/formservice.js

```diff
@@ -7,7 +7,27 @@ import App from 'js/base/app';
 
 import 'js/entities-service';
 
-const Application = App.extend({
+const ActionFormApp = App.extend({
+  beforeStart({ actionId }) {
+    return [
+      Radio.request('entities', 'fetch:forms:byAction', actionId),
+      Radio.request('entities', 'fetch:forms:definition:byAction', actionId),
+      Radio.request('entities', 'fetch:forms:fields', actionId),
+      Radio.request('entities', 'fetch:formResponses:latestSubmission', { action: actionId }),
```

Comment:
We talked about moving this to the `onStart()`. But does this work?

All this needs is the action id, correct?

Reply:
this does not work.  we need to determine based on the tags on the action what needs to be loaded.  The logic is essentially the same as https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/forms.js#L139 but instead of `form` it'd be `action`